lib/builders/latexmk.js

```diff
@@ -59,6 +60,13 @@ export default class LatexmkBuilder extends Builder {
       args.push(`-outdir="${outdir}"`)
     }
 
+    if (outputFormat) {
+      if (outputFormat !== 'pdf') {
```

Comment:
These two lines should probably be combined into
~~~ js
if (outputFormat && outputFormat !== 'pdf') {
~~~

Although since the config value is an enum, this should in reality always pass the first test, so in theory we should be able to remove the truthy check on `outputFormat`, hence reduce both if blocks down to
~~~ js
if (outputFormat !== 'pdf') {
~~~